docker-app/qfieldcloud/core/admin.py

```diff
@@ -762,6 +764,14 @@ class Meta:
         widgets = {"the_qgis_file_name": widgets.TextInput()}
         fields = "__all__"  # required for Django 3.x
 
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields["attachments_file_storage"] = forms.ChoiceField(
+            choices=get_file_storage_choices(), required=True
+        )
+        if self.instance.has_attachments_files:
```

Comment:
I was thinking to be even more conservative and count any files, not only attachments, but this should also work.